src/helpers/helpers.dom.js

```diff
@@ -56,7 +57,7 @@ function getConstraintDimension(domNode, maxStyle, percentageProperty) {
 			hasCContainer ? parseMaxStyle(constrainedContainer, parentNode, percentageProperty) : infinity);
 	}
 
-	return 'none';
+	return undefined;
```

Comment:
When the function returns undefined it looks like an error. Why not return null for example?

Reply:
@kurkle are you okay with returning `null` here?

Reply:
I'd prefer not returning anything (same as returning undefined).
There are some good arguments at [SO](https://stackoverflow.com/questions/37980559/is-it-better-to-return-undefined-or-null-from-a-javascript-function) about this.

Reply:
The most upvoted answer on the StackOverflow answer says to return `null` - though that's not the answer the asker chose. `null` makes sense to me. But I don't really care or think it matters either so I just made it an empty return for now